src/components/Form/State/State.jsx

```diff
@@ -21,6 +119,17 @@ export default class State extends ValidationElement {
     return this.props.onError(value, arr)
   }
 
+  handleUpdate(stateObj) {
+    const value = stateObj.value.length > 2
+      ? this.getStateAbbreviation(stateObj.value)
```

Comment:
I think here I would probably just do a `find` directly on `this.states`. 
You should be able to write something like this, pretty much the same code you already have:

`this.states.find((state) => state.name.toLowerCase() === stateName.toLowerCase())`

Reply:
The problem I found doing a `find` directly on `this.states` is that it doesn't take into account the additional options that can be passed through `this.props.children`.

The `MilitaryStates.jsx` component leverages the State component (`State.jsx`) and adds the Armed Forces as children.